funnel/forms/project.py

```diff
@@ -117,3 +139,15 @@ class TicketTypeForm(forms.Form):
     title = forms.StringField(__("Title"), validators=[forms.validators.DataRequired()])
     events = QuerySelectMultipleField(__("Events"),
         widget=ListWidget(), option_widget=CheckboxInput(), allow_blank=True, get_label='title', query_factory=lambda: [])
+
+
+class ProjectTicketForm(forms.Form):
+    boxoffice_data = forms.TextAreaField(__("Ticket client details"), filters=[format_json],
+        validators=[validate_json], default=BOXOFFICE_DETAILS_PLACEHOLDER)
+
+    def set_queries(self):
```

Comment:
This is misuse of `set_queries` to convert an incoming dictionary to JSON. This needs to be in a `JsonField` in Baseframe.